### ISSUE TITLE
Fix emoji functionality

### ISSUE THREAD

**Issue by mayeaux** [tinderjs/tinder-desktop#23]
Twemoji is spewing errors nonstop saying `cannot read property childNodes of undefined`

Needs to be fixed, makes dev really difficult. Honestly I'm not even sold on the twemoji module, the emojis look a little ugly/nonstandard, I'm pretty sure there is a better option which would be nice to be incorporated (we can use the same filter pattern though)

**Comment by SaturnusDJ** [comment]
I agree that the current emoji don't look good. Apple's emoji have my preference.

Maybe something usefull here: https://www.npmjs.com/search?q=emoji
https://github.com/node-modules/emoji

**Comment by mayeaux** [comment]
I just removed twemoji from the swipe page, it only errors there. Will hopefully replace this with a dif module soon
